src/utils/get-package-version.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const getPackageVersion = () => {
+  const { version } = JSON.parse(fs.readFileSync(path.resolve(__dirname, '../../package.json')));
```

Comment:
This will be the package.json of CUT? instead of the package version of the template?

Reply:
you pass the template name in but dont use it here